tests/drivers/dac/dac_loopback/boards/twr_ke18f.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&adc0 {
+	/*
+         * Increase sample time to 5 microseconds since there is no
+         * buffer between DAC and ADC.
+         */
```

Comment:
I'm guessing the CI didn't flag the spaces/tabs indentation here?

Reply:
should be fixed IMO

Reply:
Good catch, thanks! I have fixed it.